cumulus/zombienet/zombienet-sdk/tests/bootnodes/mod.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_sdk::{NetworkConfig, NetworkConfigBuilder};
+
+async fn build_network_config() -> Result<NetworkConfig, anyhow::Error> {
+	let images = zombienet_sdk::environment::get_images_from_env();
+	log::info!("Using images: {images:?}");
+	NetworkConfigBuilder::new()
+		.with_relaychain(|r| {
+			let r = r
+				.with_chain("rococo-local")
+				.with_default_command("polkadot")
+				.with_default_image(images.polkadot.as_str())
+				.with_default_args(vec!["-lparachain=debug".into(), "--no-mdns".into()])
+				// Have to set a `with_node` outside of the loop below, so that `r` has the right
+				// type.
+				.with_node(|node| node.with_name("validator-0"));
+			(1..3).fold(r, |acc, i| acc.with_node(|node| node.with_name(&format!("validator-{i}"))))
+		})
+		.with_parachain(|p| {
+			p.with_id(1000)
+				.with_default_command("polkadot-parachain")
+				.with_default_image(images.cumulus.as_str())
+				.with_chain("asset-hub-rococo-local")
+				// Do not put bootnodes into the chain-spec nor command line arguments.
+				.without_default_bootnodes()
+				// Disable mdns to rely only on DHT bootnode discovery mechanism.
+				.with_default_args(vec![
+					"-lbootnodes=trace".into(),
+					"--no-mdns".into(),
```

Comment:
nit: Should the relaychain componenta above expose `--no-mdns` as well?

Reply:
It does. Added the explanatory comment.